docs/start/installation.rst

```diff
@@ -1,14 +1,40 @@
 .. installation:
 
+Requirements
+============
+
+Photons requires a UNIX-like operating system (Linux, macOS, Windows Subsystem
+for Linux) with Python 3.6 or newer installed.
+
 Installation
 ============
 
-As long as you have a version of Python 3.6 or newer installed you can do
-something like::
+Photons is best installed in a Python virtual environment. The following commands
+create a virtual environment in the ``~/.photons-core`` directory and
+installs the latest version of Photons::
 
-    $ python3 -m venv .photons-core
-    $ source .photons-core/bin/activate
+    $ python3 -m venv ~/.photons-core
```

Comment:
I'm not sure I like suggesting it should always go in the home dir.

Reply:
Sure, but the alternative is non-deterministic, i.e. we don't know what directory the user is in before running the `python3 -m venv` command. This makes it harder to document the reactivation method.

I'll add a note that the example creates the virtualenv in the current user's home directory but this can be changed if necessary.

Reply:
I'm fine with that solution :)